assets/multiqc_config.yml

```diff
@@ -0,0 +1,14 @@
+report_comment: >
+  This report has been generated by the <a href="https://github.com/nf-core/magmap/tree/dev" target="_blank">nf-core/magmap</a> analysis pipeline. For information about how to
+  interpret these results, please see the <a href="https://nf-co.re/magmap/dev/docs/output" target="_blank">documentation</a>.
```

Comment:
Have you bumped the version with `nf-core bump-versions` ? The URL should not be `dev` for release...